cargo-pgrx/src/metadata.rs

```diff
@@ -58,5 +57,18 @@ pub fn validate(metadata: &Metadata) -> eyre::Result<()> {
         }
     }
 
+    if !mismatches.is_empty() {
+        let many = mismatches.len();
+        let mismatches = mismatches
+            .into_iter()
+            .map(|(p, v)| format!("`{p} = {v}`"))
+            .collect::<Vec<_>>()
+            .join(", ");
+
+        return Err(eyre!("The installed `cargo-pgrx` v{cargo_pgrx_version} is not compatible with the {mismatches} {} in `{}`.  `cargo-pgrx` and pgrx dependency versions must be identical.", 
```

Comment:
this line is getting very long, so it might be nice to split it up.

Reply:
```suggestion
        return Err(eyre!("The installed `cargo-pgrx` v{cargo_pgrx_version} \
        is not compatible with the {mismatches} {} in `{}`. `cargo-pgrx` \
        and pgrx dependency versions must be identical.", 
```